### PR TITLE
DM-37532: Fix meas_base imports

### PR DESCRIPTION
**Pull request by mwittgen**

### CODE REVIEW

File: python/lsst/meas/extensions/gaap/_gaap.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 import lsst.afw.table as afwTable
 import lsst.geom
 import lsst.meas.base as measBase
-from lsst.meas.base.fluxUtilities import FluxResultKey
+from lsst.meas.base import FluxResultKey
```
Comment (arunkannawadi) [inline python/lsst/meas/extensions/gaap/_gaap.py:38]: If we are going to import `FluxResultKey` from measBase, we might as well not do it and use it `FluxResultKey` with the namespace specified wherever it is used.

Comment (timj) [reply python/lsst/meas/extensions/gaap/_gaap.py:38]: Yes, the code is already importing `measBase` so may as well use `measBase.FluxResultKey` in here and remove the second import completely.

Comment (mwittgen) [reply python/lsst/meas/extensions/gaap/_gaap.py]: Implemented to `use measBase` tested locally against `meas_base` changes in 
https://github.com/lsst/meas_base/pull/233
as sell as the latest daily release.

**Review by arunkannawadi — APPROVED** [review]

**Review by erykoff — APPROVED** [review]
Looks good!

**Comment by arunkannawadi** [comment]
One minor nitpicky comment is that the commit messages should begin with capitalized letter, according to the Tim Pope's best committing practices linked from our dev guide: https://developer.lsst.io/work/flow.html#git-commit-message-best-practices

**Comment by mwittgen** [comment]
Capitalized git commit message. Will fix messages in the related pybind packages as well.

**Pull request merged**